tests-fuzz/targets/fuzz_insert_logical_table.rs

```diff
@@ -225,26 +220,62 @@ async fn execute_insert(ctx: FuzzContext, input: FuzzInput) -> Result<()> {
     });
     validator::row::assert_eq::<MySql>(&insert_expr.columns, &fetched_rows, &expected_rows)?;
 
+    Ok(())
+}
+
+async fn execute_insert(ctx: FuzzContext, input: FuzzInput) -> Result<()> {
+    info!("input: {input:?}");
+    let mut rng = ChaChaRng::seed_from_u64(input.seed);
+    let physical_table_ctx = create_physical_table(&ctx, &mut rng).await?;
+
+    let mut tables = HashMap::with_capacity(input.tables);
+
+    // Create logical tables
+    for _ in 0..input.tables {
+        let translator = CreateTableExprTranslator;
+        let create_logical_table_expr =
+            generate_create_logical_table_expr(physical_table_ctx.clone(), &mut rng).unwrap();
+        if tables.contains_key(&create_logical_table_expr.table_name) {
+            // Ignores same name logical table.
+            continue;
+        }
+        let sql = translator.translate(&create_logical_table_expr)?;
+        let result = sqlx::query(&sql)
+            .execute(&ctx.greptime)
+            .await
+            .context(error::ExecuteQuerySnafu { sql: &sql })?;
+        info!("Create logical table: {sql}, result: {result:?}");
+        let logical_table_ctx = Arc::new(TableContext::from(&create_logical_table_expr));
+
+        insert_and_validate_values(&ctx, &mut rng, logical_table_ctx.clone()).await?;
+        tables.insert(logical_table_ctx.name.clone(), logical_table_ctx);
+        if rng.gen_bool(0.1) {
+            flush_memtable(&ctx.greptime, &physical_table_ctx.name).await?;
+        }
+        if rng.gen_bool(0.1) {
+            compact_table(&ctx.greptime, &physical_table_ctx.name).await?;
+        }
```

Comment:
I think we should validate the rows in tables after flushing or compacting tables.